approval_polls/static/approval_polls/create.js

```diff
@@ -117,10 +117,8 @@ $(function () {
   })
   .on('click', 'a.remove-choice', function(e) {
     e.preventDefault();
-    console.log(e);
     var container = $(e.currentTarget).closest('.input-group');
     container.remove();
-    that.numChoiceFields--;
```

Comment:
Now that we are not decrementing the choice number, i think we should also not include the number in the placeholder text for the textfield. See https://github.com/electology/approval_frame/blob/master/approval_polls/templates/approval_polls/create.html#L39 . i.e. I believe displaying `Choice` instead of `Choice <number>` would be sufficient for the user, and we use the choice numbers internally to keep track of them.